tools/generate_keystore

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
```

make it set  -xe to fail fast

---

OK.